docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -0,0 +1,141 @@
+[[rules-ui-create]]
+[role="xpack"]
+=== Managing signal detection rules
+
+beta[]
+
+Rules run periodically and search the specified indices for 
+documents that meet their criteria.
+
+IMPORTANT:: If a rule fails to run, the {siem-app} tries to 
+rerun it at its next scheduled run time and an error is displayed in the 
+*Failure History* table on the rule's page (*SIEM* -> *Detections* ->
+*Manage signal detection rules* -> <rule name>).
+
+On the Rules page, you can:
+
+* <<load-prebuilt-rules>>
+* <<create-rule-ui, Create your own rules>>
+* <<manage-rules-ui, Clone, delete, activate, and deactivate rules>>
+* <<import-export-rules-ui>>
+
+[role="screenshot"]
+image::all-rules.png[]
+
+[[load-prebuilt-rules]]
+==== Load prebuilt Elastic rules
+
+To load the {siem-app}'s prebuilt rules, click *Load Elastic prebuilt rules* 
+on the *Signal detection rules* page (*SIEM* -> *Detections* ->
+*Manage signal detection rules*).
+
+If you delete any of the prebuilt rules, a button appears that enables 
+reloading all the deleted prebuilt rules.
+
+[NOTE]
+==============
+By default, prebuilt rules are not activated. If you want to modify a prebuilt 
+rule, you must first duplicate it and then make your changes to the duplicated rule.
+==============
+
+[[create-rule-ui]]
+==== Create a new rule
+
+. Go to *SIEM* -> *Detections* -> *Manage signal detection rules*.
+. Click *Create new rule*.
++
+The *Create new rule* page is displayed.
+[role="screenshot"]
+image::create-new-rule.png[]
+. Define which {es} indices the rule analyzes for signals.
+. Use the filter and query fields to create the criteria used for detecting 
+signals. For example, a rule that detects when a Microsoft Office product 
+starts a process in a user folder uses the following query and filters:
```

Comment:
Just noting that there are two shortcuts to supplying a query + filter, and that's either using the newer Kibana core `Saved Query` feature (by clicking the 💾),  or by clicking the `Import query from saved timeline` link at the top right of the query bar, which will present a dialog to the user for selecting an existing `Timeline` for which to grab the query+filter from. Note, this is a one-time fetching of the query+filter, and the rule's query **is not** synced/tied to the selected Timeline.